base/dcp_sharded.go

```diff
@@ -771,6 +778,7 @@ func (meh *sgMgrEventHandlers) OnFeedError(srcType string, r cbgt.Feed, feedErr
 			}
 			dcpFeed.NotifyMgrOnClose()
 		}
+		return
```

Comment:
nit: this return is kind of weird because it will just return below, i'd swap to an earlier return for readability 

```
if gone {
    return
}
```

Reply:
I agree it's a bit weird as written, but I almost introduced a bug in the previous change when I added handling below this without the return, so thought it might be a good thing to leave in place.